xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -25,8 +25,19 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Contains generic, low-level functionality for manipulating pointers.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains generic, low-level functionality for manipulating managed and unmanaged pointers.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+> [!WARNING]
+> This type is intended for advanced pointer manipulation scenarios. Callers are assumed to be familiar with ECMA-335, Sec. II.14.4 and III.1.1.5, and with the [ECMA-335 CLI Specification Addendum](https://github.com/dotnet/runtime/blob/main/docs/design/specs/Ecma-335-Augments.md).
+>
+> Most APIs on this type do not perform any argument checking or validation. Incorrect use of these APIs could destabilize the .NET runtime.
```

Comment:
```suggestion
> Most APIs on this type do not perform any argument checking or validation. Incorrect use of these APIs could lead to access violations, GC holes or destabilize the .NET runtime.
```
Came into my mind when reading this.
In essence it's similar, but to put some emphasis on the AV, and GC holes.

Reply:
Wanted to keep the Warnings section somewhat understandable by all devs, so I'll move the mention of GC holes later in the remarks section.